ROADMAP.md

```diff
@@ -0,0 +1,62 @@
+# Short Term
+(getting ready for lab days)
+
+## Outcome:
+
+- The repository covers:
+    - (rust, go, nodejs, browserjs) x (ping test)
+- We have a working demonstration and a presentation for the event
+- We have a single Dashboard to track the current state of interop testing
+
+
+## EPICs
+
+- There is a clear way to tell the state of libp2p's interoperability testing
```

Comment:
Note, in case these items are ordered by priority:

In my eyes, for the libp2p team, covering essential features (item 2) is more important than a page to discover the results (item 1). In other words, I think we should first create the things before figuring out a way to display them.